docker/setup_configuration/data.yml

```diff
@@ -0,0 +1,25 @@
+sites_config_enable: true
```

Comment:
Could you also add data for the `Selectielijst` (an extra step service in `zgw_consumers` and the config) here?